### PR TITLE
Create ap-1.md

### PR DESCRIPTION
**Pull request by ap-1**
## Describe your project in 1-3 sentences

We (@ap-1 and @EmperorParzival) are building a 3D visualizer using a 8x8x8 matrix of individually-addressable RGB LEDs. The visualizer will be programmed using FastLED on an Arduino Nano. It will be capable of a number of functions, including audio visualization and special effects. 

## Hack Club Slack handle

@anish (Anish Pallati)
@arnav (Arnav Pallati)

## Checklist before submitting proposal:

- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I am a current high school student (or younger).
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: ap-1.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+name: "@ap-1, @EmperorParzival"
```
Comment (exu3) [inline ap-1.md:2]:
```suggestion
name: ["@ap-1, @EmperorParzival"]
```

File: ap-1.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+project: "3D RGB Visualizer"
+---
+
+# Project Name
+3D RGB Visualizer
```
Comment (exu3) [inline ap-1.md:7]:
```suggestion
# 3D RGB Visualizer
```

**Review by exu3 — APPROVED** [review]
This looks great @ap-1 & @EmperorParzival! I like how detailed your plan is; it sounds like you've done thorough research on your project.

The @hackclub/bank-ops team will follow up with next steps on receiving your grant through Hack Club Bank.

Looking forward to see what you build!

**Review by MelSmith104 — COMMENTED** [review]
hey @ap-1 and @EmperorParzival ! this project sounds amazing! i love that you've chosen something new to the both of you and i think you'll have so much fun learning a new skill! im dm'ing you both on slack right now so that we can chat about next steps!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**